index.html

```diff
@@ -131,105 +131,107 @@ <h2 class="article__title">Tenant screening service</h2>
+                                <div class="agents__td td">chandersen@pixel.com</div>
+                            </div>
+                            <div class="agents__tr tr">
+                                <div class="agents__td td">Christian Andersen</div>
+                                <div class="agents__td td">Pixel Inc.</div>
+                                <div class="agents__td td">Senior</div>
+                                <div class="agents__td td">315</div>
+                                <div class="agents__td td">California</div>
+                                <div class="agents__td td">+1764156161</div>
+                                <div class="agents__td td">chandersen@pixel.com</div>
+                            </div>
                         </div>
                     </div>
-                </div>
-                <div class="agents__tfoot table">
-                    <div class="agents__tr tr">
-                        <div class="agents__th th">Total agents: 28 </div>
-                        <div class="agents__th th">
-                            <label class="results-per-side-label" for="results-per-side">Resultater per side:</label>
-                            <select id="results-per-side" name="results-per-side" class="results-per-side">
-                                <option value="6">6</option>
-                                <option value="10">10</option>
-                                <option value="15">15</option>
-                                <option value="20">20</option>
-                            </select>
-                        </div>
-                        <div class="agents__th th">
-                            <div class="agents__pagination">
-                                <div class="agents__pagination-item current">
-                                    Side
-                                    <span class="agents__pagination-text">1</span>
-                                </div>
-                                <div class="agents__pagination-item">
-                                    <span class="agents__pagination-text">av</span>
-                                </div>
-                                <div class="agents__pagination-item total">
-                                    <span class="agents__pagination-text">5</span>
-                                </div>
-                                <div class="agents__pagination-controls">
-                                    <a href="#next" class="agents__pagination-prev agents__pagination-arrow inactive"></a>
-                                    <a href="#prev" class="agents__pagination-next agents__pagination-arrow"></a>
-                                </div>
+                    <div class="agents__tfoot agents__footer table">
+                        <div class="agents__tr tr">
+                            <div class="agents__th th">Total agents: 28 </div>
```

Comment:
Texts and values are better to wrap separately. That helps a lot for JS actions (clone, insert, get value) where you won't need any parsing.